aw_watcher_window/macos.py

```diff
@@ -1,17 +1,28 @@
-import subprocess
-from subprocess import PIPE
-import os
+from typing import Optional
+from AppKit import NSWorkspace
+from Quartz import (
+    CGWindowListCopyWindowInfo,
+    kCGWindowListOptionOnScreenOnly,
+    kCGNullWindowID
+)
 
+def getInfo() -> Optional[dict]:
+    app_name = ''
+    title = ''
 
-def getInfo() -> str:
-    cmd = ["osascript", os.path.join(os.path.dirname(os.path.realpath(__file__)), "printAppTitle.scpt")]
-    p = subprocess.run(cmd, stdout=PIPE)
-    return str(p.stdout, "utf8").strip()
+    app = NSWorkspace.sharedWorkspace().activeApplication()
```

Comment:
`activeApplication` is deprecated: https://developer.apple.com/documentation/appkit/nsworkspace/1530411-activeapplication, 

and is replaced by `frontmostApplication`: https://developer.apple.com/documentation/appkit/nsworkspace/1532097-frontmostapplication